.github/workflows/issues.yaml

```diff
@@ -1,17 +0,0 @@
-# Uses https://github.com/actions/add-to-project example to add issues to Anchore OSS project
```

Comment:
Deleting this file seems unrelated

Reply:
Yep! It's not doing anything right now so was cleaning up where I could while I was in the workflows file.